src/OrchardCore.Modules/OrchardCore.Menu/Views/MenuItemPermissionPart.Edit.cshtml

```diff
@@ -7,54 +9,55 @@
 }
 
 <script asp-name="vuejs" at="Foot" version="2"></script>
-<script asp-name="vue-multiselect" at="Foot"></script>
-<script asp-name="menu-permission-picker" asp-src="~/OrchardCore.Menu/Scripts/menu-permission-picker.min.js" debug-src="~/OrchardCore.Menu/Scripts/menu-permission-picker.js" at="Foot" depends-on="vuejs:2,vue-multiselect"></script>
-<style asp-name="menu-permission-picker" asp-src="~/OrchardCore.Menu/Styles/menu-permission-picker.min.css" debug-src="~/OrchardCore.Menu/Styles/menu-permission-picker.css" depends-on="vue-multiselect"></style>
-
-<div class="mb-3">
+<script asp-name="vue-multiselect" version="2" at="Foot"></script>
```

Comment:
You can remove both of these lines:

```
<script asp-name="vuejs" at="Foot" version="2"></script>
<script asp-name="vue-multiselect" version="2" at="Foot"></script>

```

It will be equivalent.

Reply:
agreed